canonical-web-and-design/maas-ui
# style(ui): fix no-redeclare warnings

## Done

- Update slices to use unique types and slice names.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run `dotrun ui`.
- You will see a bunch of warnings (that hopefully will be fixed with our dep updates).
- Check that there are no warnings like this:
```
[0] ./src/app/store/zone/slice.ts
[0]   Line 10:7:  'ZoneSlice' is already defined  no-redeclare
```

## Fixes

Fixes: canonical-web-and-design/maas-ui#1692
